content/02.introduction.md

```diff
@@ -1,10 +1,24 @@
 ## Introduction
-1. What is a preprint
-2. Why are preprints important?
-3. Mention how preprints are being integrated into scientist’s everyday workflow
 
-1. Talk about biorxiv and discuss how it is one of the repositories that maintains preprints along with citation of others such as arxiv/medrxiv etc.
-2. Discuss works that analyze biorxiv from an audience perspective (quantifying tweets etc.)
-3. Mention the gap which consists of analysing the language of biorxiv preprints (first to do this)
-4. ^ Why is this important? What will this allow for future research projects?
-5. Provide list of contributions within this manuscript
+Preprints are scholarly works that are shared before they have been formally peer reviewed and published.
+The practice of sharing manuscripts before peer review has a long history [@doi:10.1371/journal.pbio.2003995].
+The longest use on an ongoing basis started with physicists in the 1990s [@doi:10.15252/embj.201695531] and have become an essential tool for the scientific community [@doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035; @doi:10.1126/science.aaf9133; @doi:10.1038/s41557-020-0477-5].
+Preprints within the life sciences community have faced years of backlash from publishers [@doi:10.1371/journal.pbio.2003995], but have made a resurgence within the past decade [@doi:10.1101/833400; @doi:10.7554/eLife.45133.001].
+Common preprint repositories include arXiv [@doi:10.1108/14666180010345564], bioRxiv [@doi:10.1101/833400] and medRxiv [@url:https://www.medrxiv.org/]; however, there are over 60 different preprint repositories [@url:https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/].
+Overall, preprints are becoming widely accepted within the life sciences community [@doi:10.1371/journal.pmed.1002549; @doi:10.1038/530265a; @doi:10.1371/journal.pbio.3000151; @doi:10.1099/mgen.0.000259].
+
+The scientific community has begun to analyze the impact of preprints in the life sciences.
+Studies have found that preprints are being posted at a high rate [@doi:10.1371/journal.pbio.3000269] and are rapidly shared on social media, routinely downloaded, and cited often [@doi:10.1371/journal.pone.0047523].
+Certain categories of preprints seem to be read and shared differently by both scientists and non-scientists [@doi:10.1101/2020.03.06.981589].
+Across preprint servers, analyses suggest that between two thirds and three quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/pra2.175].
+Studies have found that preprints often lower the amount of time required to be published in a journal [@doi:10.2139/ssrn.3455146].

```

Comment:
>Whatever drives the need to update preprints, there is a benefit of sorts: the time from latest available preprint to publication is over three weeks shorter than that between first preprint submission and publication (Figure 2). But when we add in time spent working on additional versions (7.7 weeks median average) — overall, preprints with just one version on bioRxiv are published fastest. 

Figure 2 shows that preprints with multiple versions take the lowest time to get published, but when they factor in the amount of time between versions it is higher. Now that you mention it a figure to show this would have been nice.

Reply:
If I'm reading this correctly, it doesn't say that papers with preprints get published faster than those without though. I read the sentence as saying that. Did they do any analyses of papers that lacked preprints?

Reply:
It's also possible I'm misreading what the sentence in our paper is asserting.

Reply:
I see now. I'll edit that sentence to be more clear.